SingularityUI/app/utils.es6

```diff
@@ -45,6 +46,10 @@ const Utils = {
     ));
   },
 
+  humanizeSlaveHostName(longHostName) {
```

Comment:
I'll check out how this looks live in the ui, but this might be a bit specific to how hubspot does naming. Most are probably going to want to see the full hostname